tools/ui-components/src/image/image.types.tsx

```diff
@@ -0,0 +1,5 @@
+export interface ImageProps {
```

Comment:
Nit: Since this file does not contain components, we can just use the `.ts` extension instead.